website/docs/getting-started/networks.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: Networks
+---
+
+:::note
+This document details the configuration and use of Tezos testnets. If you are looking for information on running or deploying to a sandbox, please see the Flextesa documentation [here](../docs/plugins/plugin-flextesa.md/))
+:::
+
+## Introduction
+
+Taqueria provides support to interact with Tezos networks in a simplified and streamlined manner. Taqueria provides a centralized configuration for testnets (including faucet keys) which can then be targeted by Taqueria through the use of Taqueria Environments
```

Comment:
Is 'Taqueria Environments' defined somewhere else as a concept? If not, we should start to introduce this as a core concept in the beginner docs.

Reply:
It's a separate doc that's next on the list